mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -205,6 +206,69 @@ class ALSSuite
     assert(decompressed.toSet === expected)
   }
 
+  test("CheckedCast") {
+    val checkedCast = new ALS().checkedCast
+    val df = spark.sqlContext.range(1)
+
+    withClue("Valid Integer Ids") {
+      df.select(checkedCast( lit(123) )).collect()
```

Comment:
Could you remove the spaces around `lit(123)` here and throughout for code style?